pegasus/sites.v3/code.org/views/elementary_school_curricula_video.haml

```diff
@@ -1,37 +1,40 @@
-- video_curricula_index = 5
+- show_game_design_curriculum = DCDO.get('curriculum-launch-2024', false)

```

Comment:
I think this might need the `!!` operator before the DCDO flag statement.
```suggestion
- show_game_design_curriculum = !!DCDO.get('curriculum-launch-2024', false)

```